runtime/runtime/src/lib.rs

```diff
@@ -167,6 +167,22 @@ impl Runtime {
         }
     }
 
+    /// Subtracts the storage rent from the given account balance.
+    fn apply_rent(account_id: &AccountId, account: &mut Account, block_index: BlockIndex) {
```

Comment:
yeah, going forward this should probably return serialized account and key and we can just call non serializable `set` on that.

Reply:
Will clean up redundant computations after I remove mana and gas.